README.md

```diff
@@ -16,30 +16,61 @@
   * [Join the Gitter chat room](https://gitter.im/cocotb)
 
 ## Installation
+Cocotb can be installed by running `pip install cocotb`, and requires:
 
-Cocotb can be installed by running `pip install cocotb`.
+- Python 3.5+
+- A C++ compiler
+- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/))
 
-## Quickstart
+For more details see [the documentation](https://cocotb.readthedocs.io/en/latest/quickstart.html#pre-requisites).
 
-    # Install pre-requisites (waveform viewer optional)
-    sudo yum install -y iverilog python-devel gtkwave
+## Usage
 
```

Comment:
Start with a sentence of what you're trying to show here, e.g.

As first trivial introduction to cocotb, the following example "tests" a flip-flop.